subprojects/workers/src/main/java/org/gradle/workers/WorkerExecutor.java

```diff
@@ -55,8 +56,23 @@
      * in the {@link WorkerConfiguration}.  If no idle daemons are available, a new daemon will be started.  Any errors
      * will be thrown from {@link #await()} or from the surrounding task action if {@link #await()} is not used.
      */
+    @Deprecated
     void submit(Class<? extends Runnable> actionClass, Action<? super WorkerConfiguration> configAction);
```

Comment:
Not for 5.6, but we should put this on a list somewhere for deprecations to add to 6.0 so we can consider removing this in 7.0 (or hiding it more)